src/core/public/chrome/ui/header/branding/opensearch_dashboards_logo_darkmode.tsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+import React from 'react';
+
+export function OpenSearchDashboardsLogoDarkMode() {
+  return (
```

Comment:
I think for draft purposes it's probably like this but we can probably be consistent with the other svgs were an svg file was created and accessed in the react component (without the helper functions):

```
addBasePath(
 `/plugins/${PLUGIN_ID}/assets/opensearch_dashboards_${appId}_${
   IS_DARK_THEME ? 'dark' : 'light'
   }.svg`
 )}
  ```
  
  then we just make this opensearch_dashboards_logo.tsx

Reply:
Are you saying that this code would go in `header_logo.tsx`? Where do the `PLUGIN_ID`, `appId`, and `IS_DARK_THEME` props get passed in to the component from?

Reply:
And how do I test enabling/disabling dark mode?

Reply:
In order to get these changes out for 1.0, I've added a follow-up task to the meta issue to address this feature for darkmode/default logos based on the theme: https://github.com/opensearch-project/OpenSearch-Dashboards/issues/372